variables.tf

```diff
@@ -51,6 +51,6 @@ variable "feature_gates" {
 }
 
 variable "calico_enabled" {
-  default = false
+  default = true
```

Comment:
This would introduce a behaviour change. I think it's better to stay with the old behaviour.